zio-http/src/test/scala/zhttp/internal/HttpGen.scala

```diff
@@ -93,6 +96,10 @@ object HttpGen {
     ),
   )
 
+  def scheme: Gen[Any, Scheme]                    = Gen.fromIterable(List(HTTP, HTTPS, WS, WSS))
+  def jHttpScheme: Gen[Any, HttpScheme]           = Gen.fromIterable(List(HttpScheme.HTTP, HttpScheme.HTTPS))
+  def jWebSocketScheme: Gen[Any, WebSocketScheme] = Gen.fromIterable(List(WebSocketScheme.WS, WebSocketScheme.WSS))
```

Comment:
Let's move the netty gens close to the tests and move it out of this file. This file is only for ZIO Http Gens.